store/types/utils.go

```diff
@@ -105,3 +108,16 @@ func PrefixEndBytes(prefix []byte) []byte {
 func InclusiveEndBytes(inclusiveBytes []byte) []byte {
 	return append(inclusiveBytes, byte(0x00))
 }
+
+// assertNoCommonPrefix will panic if there are two keys: k1 and k2 in keys, such that
+// k1 is a prefix of k2
+func assertNoPrefix(keys []string) {
```

Comment:
`assertNoCommonPrefix` makes more sense to me rather than `assertNoPrefix`

Reply:
good call, thanks